source/common/dynamo/dynamo_filter.cc

```diff
@@ -35,6 +35,7 @@ Http::FilterDataStatus DynamoFilter::decodeData(Buffer::Instance& data, bool end
   if (end_stream) {
     return Http::FilterDataStatus::Continue;
   } else {
+    // Buffer until the complete request has been processed.
```

Comment:
Sorry I wasn't referring to this (comment is fine with me if you want it), but the extra boilerplate functions every filter needs to implement, which I think we have agreed will now be opt-in. 👍